docs/content/en/integrations.md

```diff
@@ -284,3 +284,22 @@ We recommend to take a look at this tutorial made by [Pascal Cauhépé](https://
 👉 &nbsp;https://nuxt-content-and-forestry.netlify.app
 
 </alert>
+
+## @nuxtjs/i18n
```

Comment:
Package name is (yet) not `@nuxtjs/i18n`). Maybe we can simply use `Localization` for title?

Reply:
Sounds good to me. Do you prefer `Localization`, `Internationalization` or simply `nuxt-i18n` for the title?

Reply:
While normally I'd agree that `Localization` is the name to go with, the module referenced directly after has `i18n` in the name. Perhaps the title could be both `Localization & Internationalization`?